packages/http-client/src/http-client.ts

```diff
@@ -133,6 +135,7 @@ export function execute(executeFn: ExecuteHttpRequestFn<HttpResponse>) {
         jwt: destination.jwt,
         requestConfig: request,
         uri: resolvedDestination.url,
+        destinationName: resolvedDestination.name ?? undefined,
```

Comment:
Not fully clear to me how this is different from omitting the `?? undefined` part. Is this about null vs undefined?

Reply:
Seems like that has to be the reason... but I am surprised that `resolvedDestination.name` would ever be `null`

Reply:
yes, I wanted to remove the `null` in the types of the destination but I decided not to do that in this PR. Perhaps the destination service returns objects with `null` value in the JSON?